scripts/policy/misc/unknown-protocols.zeek

```diff
@@ -0,0 +1,58 @@
+##! This script logs information about packet protocols that Zeek doesn't
+##! know how to process. Mostly these come from packet analysis plugins when
+##! they attempt to forward to the next analyzer, but they also can originate
+##! from non-packet analyzers.
+
+@load base/frameworks/notice
+
+module UnknownProtocol;
+
+export {
+	redef enum Log::ID += { LOG };
+
+	global log_policy: Log::PolicyHook;
+
+	type Info: record {
+		## Timestamp for when the measurement occurred.
+		ts:           time     &log;
+		##
+		analyzer:     string   &log;
+		protocol_id:  string   &log;
+		first_bytes:  string   &log;
+	};
+
+	## How many reports for an analyzer/protocol pair will be allowed to
+	## raise events for logging being rate-limited.
+	option sampling_threshold : count = 25 &redef;
+
+	## The rate-limiting sampling rate. One out of every of this number of
+	## rate-limited pairs of a given type will be allowed to raise events
+	## for further script-layer handling. Setting the sampling rate to 0
+	## will disable all output of rate-limited pairs.
+	option sampling_rate : count = 1000 &redef;
```

Comment:
Again, thinking we may want pretty generous defaults since this stuff is all possibly checking per-packet.  Maybe 100k ?